package/src/skia/Data/Data.ts

```diff
@@ -0,0 +1,46 @@
+import type { DependencyList } from "react";
+import { useEffect, useState } from "react";
+
+import type { SkJSIInstance } from "../JsiInstance";
+import { Skia } from "../Skia";
+
+export type Data = SkJSIInstance<"Data">;
+
+const resolveAsset = require("react-native/Libraries/Image/resolveAssetSource");
```

Comment:
I know we've been using this through the require previously as well - but after checking out the documentation is looks like RN has this method exposed on the `Image` component [directly](https://reactnative.dev/docs/image#resolveassetsource). Doing this require might break when RN updates.